### PR TITLE
Added support for PHP REPL

### PR DESCRIPTION
**Pull request by AW3i**
The REPL used for PHP is psysh

### CODE REVIEW

File: autoload/codi/load.vim [hunk 1]
```diff
@@ -67,6 +67,10 @@ let s:codi_default_interpreters = {
           \ 'bin': ['planck', '--verbose', '--dumb-terminal'],
           \ 'prompt': '^.\{-}=> ',
           \ },
+      \ 'php': {
+          \ 'bin': ['psysh'],
+          \ 'prompt': '^\(>>>\|\.\.\.\) ',
```
Comment (metakirby5) [inline autoload/codi/load.vim]: Can you add a `preprocess` to get rid of the fat arrow from `psysh` output? It should look pretty much the same as `s:pp_r`. In fact, you might just want to rename the function to `s:pp_fat_arrow` and reuse it.

**Comment by metakirby5** [comment]
Thanks!

**Comment by still-dreaming-1** [comment]
Sweet, I was going to add this at some point!

**Comment by mblarsen** [comment]
Multi-line output like e.g an array shows only the first line `[` @AW3i

**Comment by metakirby5** [comment]
@mblarsen Codi generally has trouble with multi-line output, as it still needs to maintain alignment with the other buffer. This is related to #27

**Comment by fcpg** [comment]
Any chance to hack some workaround for this? Not getting array results strongly lowers the interest of this otherwise fine tool, as far as PHP is concerned. Perhaps you could just join() the array, or create Vim folds.

**Comment by metakirby5** [comment]
@fcpg I don't know much about PHP, but you're welcome to give your idea a shot. See `:h codi-interpreters-rephrase`.

**Pull request merged**